src/CollectionsFrameWorkChallenge/ArrayListChallenge.java

```diff
@@ -0,0 +1,47 @@
+package CollectionsFrameWorkChallenge;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class ArrayListChallenge {
+    public static void main(String[] args) {
+        List<String> colorList = new ArrayList<>();
+        colorList.add("Red");
+        colorList.add("Blue");
+        colorList.add("Green");
+        colorList.add("Black");
+        System.out.println(colorList);
+        for (String element : colorList) {
+            System.out.println(element);
+        }
+        colorList.add(0, "Purple");
+        colorList.remove(3);
+        colorList.set(2, "Pink");
+        colorList.remove(2);
+        colorList.contains("Red"); // evaluate returns true
+        Collections.sort(colorList);
+        List<String> copiedList = new ArrayList<>(colorList); // or copiedList.addAll(colorList);
+        Collections.shuffle(copiedList);
+        Collections.reverse(colorList);
+        System.out.println(colorList.subList(0, 1));
+        colorList.equals(copiedList); // evaluate returns false
+        Collections.swap(colorList, 0, 1);
+        List<String> carList = Arrays.asList("Kia", "Bmw", "Logan");
+        colorList.addAll(carList);
+        List<String> cloneList = new ArrayList<>(colorList);
```

Comment:
Not the most intuitive name. We have a clone list of what? Can we add in the name maybe something related to the colorList?